ejector/ejector_config.go

```diff
@@ -116,6 +127,28 @@ func (e *RootEjectorConfig) Verify() error {
 	return nil
 }
 
+var _ config.VerifiableConfig = (*EjectorSecretConfig)(nil)
+
+// Configuration for secrets used by the ejector.
+type EjectorSecretConfig struct {
+	// The Ethereum RPC URL(s) to use for connecting to the blockchain.
+	EthRpcUrls []string `docs:"required"`
+
+	// The private key to use for signing ejection transactions, in hex.
+	// Do not include the '0x' prefix.
```

Comment:
Does `0x` actually break stuff? Normally hex parsing utils are `0x` tolerant

Reply:
I believe it does... some of the libraries and utilities we use require the `0x`, and others require it to be absent. It's really quite confusing.